HARK/distribution.py

```diff
@@ -616,9 +715,9 @@ def approx(self, N, equiprobable=False):
 
         # Now find a discretization for a univariate standard normal.
         if equiprobable:
-            z_approx = Normal().approx_equiprobable(N)
+            z_approx = Normal().discretize(N, method="equiprobable")
```

Comment:
I'm a little confused by the structure here...

`_approx` as a private method (?) which takes a boolean argument that corresponds to a string variable to a public method... Seems like we would want `method` instead of equiprobable here and just pass the string through.

Reply:
That makes sense. I'll change this.